@claude

fix the following errors:

```
____________________________ test_EditReferenceText ____________________________
db = None
    def test_EditReferenceText(db):
        super_user, super_token = generate_user(is_superuser=True)
        test_new_text = "New Text"
    
        user1, user1_token = generate_user()
        user2, user2_token = generate_user()
    
        with session_scope() as session:
            with references_session(user1_token) as api:
>               reference = api.WriteFriendReference(
                    references_pb2.WriteFriendReferenceReq(
                        to_user_id=user2.id, text="Old Text", private_text="", was_appropriate=True, rating=1
                    )
                )
src/tests/test_admin.py:521: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/tests/test_fixtures.py:796: in fake_handler
    response = handler.unary_unary(request, context, session)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/couchers/servicers/references.py:240: in WriteFriendReference
    context.abort_with_error_code(grpc.StatusCode.FAILED_PRECONDITION, "can_only_refer_friends")
src/couchers/context.py:142: in abort_with_error_code
    context.abort(status_code, error_message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <tests.test_fixtures.MockGrpcContext object at 0x7e3942a0d0d0>
code = <StatusCode.FAILED_PRECONDITION: (9, 'failed precondition')>
details = 'You can only write friend references for confirmed friends.'
    def abort(self, code, details):
>       raise FakeRpcError(code, details)
E       tests.test_fixtures.FakeRpcError: (<StatusCode.FAILED_PRECONDITION: (9, 'failed precondition')>, 'You can only write friend references for confirmed friends.')
src/tests/test_fixtures.py:738: FakeRpcError
_____________________________ test_DeleteReference _____________________________
db = None
    def test_DeleteReference(db):
        super_user, super_token = generate_user(is_superuser=True)
    
        user1, user1_token = generate_user()
        user2, user2_token = generate_user()
    
        with references_session(user1_token) as api:
>           reference = api.WriteFriendReference(
                references_pb2.WriteFriendReferenceReq(
                    to_user_id=user2.id, text="Old Text", private_text="", was_appropriate=True, rating=1
                )
            )
src/tests/test_admin.py:547: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/tests/test_fixtures.py:796: in fake_handler
    response = handler.unary_unary(request, context, session)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/couchers/servicers/references.py:240: in WriteFriendReference
    context.abort_with_error_code(grpc.StatusCode.FAILED_PRECONDITION, "can_only_refer_friends")
src/couchers/context.py:142: in abort_with_error_code
    context.abort(status_code, error_message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <tests.test_fixtures.MockGrpcContext object at 0x7e3942146c30>
code = <StatusCode.FAILED_PRECONDITION: (9, 'failed precondition')>
details = 'You can only write friend references for confirmed friends.'
    def abort(self, code, details):
>       raise FakeRpcError(code, details)
E       tests.test_fixtures.FakeRpcError: (<StatusCode.FAILED_PRECONDITION: (9, 'failed precondition')>, 'You can only write friend references for confirmed friends.')
src/tests/test_fixtures.py:738: FakeRpcError
```

--- a/app/backend/src/tests/test_admin.py
+++ b/app/backend/src/tests/test_admin.py
@@ -26,6 +26,7 @@ from tests.test_fixtures import (  # noqa
     events_session,
     generate_user,
     get_user_id_and_token,
+    make_friends,
     mock_notification_email,
     push_collector,
     real_admin_session,
@@ -515,6 +516,7 @@ def test_EditReferenceText(db):
 
     user1, user1_token = generate_user()
     user2, user2_token = generate_user()
+    make_friends(user1, user2)
 
     with session_scope() as session:
         with references_session(user1_token) as api:
@@ -542,6 +544,7 @@ def test_DeleteReference(db):
 
     user1, user1_token = generate_user()
     user2, user2_token = generate_user()
+    make_friends(user1, user2)
 
     with references_session(user1_token) as api:
         reference = api.WriteFriendReference(

